src/App.js

```diff
@@ -31,6 +31,16 @@ class App extends Component {
       .catch(error => console.log(error));
   }
 
+  updateCurrentSetlist = (id) => {
+    let show = this.state.concertData.find((concert) => {
```

Comment:
Could probably even cut out the show variable and give currentShow the return value of the find directly. But that's more just nitpicky refactoring things..